db/re/item_db.txt

```diff
@@ -11969,23 +12157,112 @@
+25017,MATKStone_Bottom,MATK Stone(Lower),6,,,100,,,,,,,,,,,,,{ bonus bMatkRate,1; },{},{}
 25043,Thorny_Vine_Flute,Thorny Vine Flute,3,0,,0,,,,,,,,,,,,,{},{},{}
 25044,Hard_Thorny_Vine,Hard Thorny Vine,3,0,,10,,,,,,,,,,,,,{},{},{}
 25045,Luxurious_Cloth,Luxurious Cloth,3,0,,0,,,,,,,,,,,,,{},{},{}
 25046,Boarding_Pass,Boarding Pass,3,0,,0,,,,,,,,,,,,,{},{},{}
 25047,Kahlunac,Kahlunac,3,0,,0,,,,,,,,,,,,,{},{},{}
 25048,Hearty_Lunchbox,Hearty Lunchbox,3,0,,0,,,,,,,,,,,,,{},{},{}
 25049,Basilac_Clam,Basilac Clam,3,0,,0,,,,,,,,,,,,,{},{},{}
+25058,TwinkleEffect_Top,TwinkleEffect (Upper),6,,,100,,,,,,,,,,,,,{/*TODO*/},{},{}
+25059,GhostEffect_Middle,GhostEffect (Middle),6,,,100,,,,,,,,,,,,,{/*TODO*/},{},{}
+25060,Critical_Stone,Critical Stone (Middle),6,,,100,,,,,,,,,,,,,{ bonus bCritAtkRate,3; },{},{}
+25061,Range_Stone,Range Stone (Middle),6,,,100,,,,,,,,,,,,,{ bonus bLongAtkRate,3; },{},{}
+25062,Greed_Stone,Greed Stone (Bottom),6,,,100,,,,,,,,,,,,,{ skill "BS_GREED",1; },{},{}
+25063,MaxHP2_Stone,MaxHP2 Stone (Bottom),6,,,100,,,,,,,,,,,,,{ bonus bMaxHP,100; if (BaseLevel >= 120) bonus bMaxHPrate,2; },{},{}
+25064,MaxSP2_Stone,MaxSP2 Stone (Bottom),6,,,100,,,,,,,,,,,,,{ bonus bMaxSP,50; if (BaseLevel >= 120) bonus bMaxHPrate,2; },{},{}
+25065,Detoxify_Stone,Detoxify Stone (Bottom),6,,,100,,,,,,,,,,,,,{ skill "TF_DETOXIFY",1; },{},{}
+25066,Recovery_Stone,Recovery Stoen (Bottom),6,,,100,,,,,,,,,,,,,{ skill "PR_STRECOVER",1; },{},{}
+25067,CastingStone_Robe,CastingStone (Garment),6,,,100,,,,,,,,,,,,,{ bonus bFixedCast,-500; },{},{}
+25068,ASPDStone_Top,ASPDStone (Upper),6,,,100,,,,,,,,,,,,,{ bonus bAspd,1; },{},{}
+25069,ReloadStone_Top,ReloadStone (Upper),6,,,100,,,,,,,,,,,,,{ bonus bDelayrate,-1; },{},{}
+25070,ReloadStone_Middle,ReloadStone (Middle),6,,,100,,,,,,,,,,,,,{ bonus bDelayrate,-1; },{},{}
+25071,ReloadStone_Bottom,ReloadStone (Bottom),6,,,100,,,,,,,,,,,,,{ bonus bDelayrate,-1; },{},{}
+25072,Kyrie_Stone,Kyrie Stone (Bottom),6,,,100,,,,,,,,,,,,,{ skill "PR_KYRIE",1; },{},{}
 25127,Silent_Energy_Particle,Silent Energy Particle,3,0,,0,,,,,,,,,,,,,{},{},{}
 25128,Weak_Energy_Particle,Weak Energy Particle,3,0,,0,,,,,,,,,,,,,{},{},{}
 25129,Unstable_Energy_Particle,Unstable Energy Particle,3,0,,0,,,,,,,,,,,,,{},{},{}
 25130,Sinister_Energy_Particle,Sinister Energy Particle,3,0,,0,,,,,,,,,,,,,{},{},{}
 25131,Fallen_Energy_Particle,Fallen Energy Particle,3,0,,0,,,,,,,,,,,,,{},{},{}
 25132,Pumpkin_Deco,Pumpkin Deco,3,0,,0,,,,,,,,,,,,,{},{},{}
 25133,Dried_White_Stem,Dried White Stem,3,0,,0,,,,,,,,,,,,,{},{},{}
+25136,ElectricEffect_Middle,ElectricEffect (Middle),6,,,100,,,,,,,,,,,,,{ hateffect HAT_EF_Electric,true; },{},{ hateffect HAT_EF_Electric,false; }
+25137,GreenFloor_Bottom,GreenFloor (Bottom),6,,,100,,,,,,,,,,,,,{ hateffect HAT_EF_Green_Floor,true; },{},{ hateffect HAT_EF_Green_Floor,false; }
+25138,ShrinkEffect_Middle,ShrinkEffect (Middle),6,,,100,,,,,,,,,,,,,{/*TODO*/},{},{}
```

Comment:
This effect in KRO make player size to mini-sprite

It should be "HAT_EF_Shrink"

https://www.youtube.com/watch?v=0X-0Hoq-QaQ&t=16s

See Archbishop in 0.21